packages/harmony/src/components/typography/Text/types.ts

```diff
@@ -22,7 +22,7 @@ export type TextProps<TextComponentType extends ElementType = 'p'> =
 
 export type TextStrength = 'weak' | 'default' | 'strong'
 
-export type TextColor = 'heading' | 'default' | 'subdued' | 'disabled'
+export type TextColor = 'heading' | 'default' | 'subdued' | 'disabled' | 'error'
```

Comment:
@Kyle-Shanks Added the `error` text color 👀

Reply:
nice! `danger` or something more cover more cases like destructive button text which isn't an error, but this works.

Reply:
I just followed whatever Sammie did on the design side, just asked about `danger` instead
https://audius-internal.slack.com/archives/C05MUGKK4AG/p1697499291702829?thread_ts=1697129541.855729&cid=C05MUGKK4AG

Reply:
This was updated to `danger` on both the design side and here 👍